TestCafe spinner doesn't stop in live mode if a test file has syntax error

<!--
If you have all reproduction steps with a complete sample app, please share as many details as possible in the sections below.

Make sure that you tried using the latest TestCafe version (https://github.com/DevExpress/testcafe/releases), where this behavior might have been already addressed.

Before submitting an issue, please check CONTRIBUTING.md and existing issues in this repository (https://github.com/DevExpress/testcafe/issues) in case a similar issue exists or was already addressed. This may save your time (and ours).
-->

### What is your Test Scenario?
Make a mistake in a test file and run it in live mode.
 
### What is the Current behavior?
TestCafe doesn't hide the spinner after printing an error. 
```
F:\test\release> npx testcafe -L
|

Live mode is enabled.
TestCafe now watches source files and reruns
the tests once the changes are saved.

You can use the following keys in the terminal:
'Ctrl+S' - stops the test run;
'Ctrl+R' - restarts the test run;
'Ctrl+W' - enables/disables watching files;
'Ctrl+C' - quits live mode and closes the browsers.


Watching the following files:
  F:\test\release\blabla\testi.ts
|

Make changes to the source files or press Ctrl+R to restart the test run.
ERROR: Error: Cannot prepare tests due to an error.

Error: TypeScript compilation failed.
F:/test/release/blabla/testi.ts (5, 20): Cannot find module './blablabla'.
/       <= spins until <Ctrl> + R is pressed or the test file is edited

```

### What is the Expected behavior?
TestCafe should hide the spinner after printing an error.

--- a/src/runner/index.js
+++ b/src/runner/index.js
@@ -269,7 +269,13 @@ export default class Runner extends EventEmitter {
     }
 
     _createRunnableConfiguration () {
-        return this.bootstrapper.createRunnableConfiguration();
+        return this.bootstrapper
+            .createRunnableConfiguration()
+            .then(runnableConfiguration => {
+                this.emit('done-bootstrapping');
+
+                return runnableConfiguration;
+            });
     }
 
     _validateScreenshotPath (screenshotPath, pathType) {
@@ -425,8 +431,6 @@ export default class Runner extends EventEmitter {
             .then(() => this._validateRunOptions())
             .then(() => this._createRunnableConfiguration())
             .then(({ reporterPlugins, browserSet, tests, testedApp }) => {
-                this.emit('done-bootstrapping');
-
                 return this._runTask(reporterPlugins, browserSet, tests, testedApp);
             });
 

--- a/test/server/data/test-suites/live/test-external-unexisting-module.js
+++ b/test/server/data/test-suites/live/test-external-unexisting-module.js
@@ -1,0 +1,7 @@
+require('./non-existing-module');
+
+fixture('Fixture5')
+    .page('http://example.com');
+
+test('test6', async t => {
+});

--- a/test/server/live-test.js
+++ b/test/server/live-test.js
@@ -8,10 +8,11 @@ const LiveModeController   = require('../../lib/live/controller');
 const LiveModeRunner       = require('../../lib/live/test-runner');
 const LiveModeBootstrapper = require('../../lib/live/bootstrapper');
 
-const testFileWithSingleTestPath     = path.resolve('test/server/data/test-suites/live/test.js');
-const testFileWithMultipleTestsPath  = path.resolve('test/server/data/test-suites/live/multiple-tests.js');
-const testFileWithSyntaxErrorPath    = path.resolve('test/server/data/test-suites/live/test-with-syntax-error.js');
-const testFileWithExternalModulePath = path.resolve('test/server/data/test-suites/live/test-external-module.js');
+const testFileWithSingleTestPath               = path.resolve('test/server/data/test-suites/live/test.js');
+const testFileWithMultipleTestsPath            = path.resolve('test/server/data/test-suites/live/multiple-tests.js');
+const testFileWithSyntaxErrorPath              = path.resolve('test/server/data/test-suites/live/test-with-syntax-error.js');
+const testFileWithExternalModulePath           = path.resolve('test/server/data/test-suites/live/test-external-module.js');
+const testFileWithExternalUnexistingModulePath = path.resolve('test/server/data/test-suites/live/test-external-unexisting-module.js');
 
 const externalModulePath = path.resolve('test/server/data/test-suites/live/module.js');
 
@@ -74,13 +75,15 @@ class BootstrapperMock extends LiveModeBootstrapper {
 }
 
 class RunnerMock extends LiveModeRunner {
-    constructor ({ proxy, browserConnectionGateway, configuration }, { runTimeout = 0, errorOnValidate = false }) {
+    constructor ({ proxy, browserConnectionGateway, configuration }, { runTimeout = 0, errorOnValidate = false, onBootstrapDone = noop }) {
         super(proxy, browserConnectionGateway, configuration.clone());
 
         this.runCount        = 0;
         this.runTimeout      = runTimeout;
         this.errorOnValidate = errorOnValidate;
         this.disposed        = false;
+
+        this.once('done-bootstrapping', onBootstrapDone);
     }
 
     get watchedFiles () {
@@ -278,6 +281,21 @@ describe('TestCafe Live', function () {
             .then(() => {
                 expect(errors.length).eql(1);
                 expect(errors[0].toString()).contains('Error: Cannot prepare tests due to an error');
+            });
+    });
+
+    it('done-bootstrap event', function () {
+        let handled = false;
+
+        return runTests(testFileWithExternalUnexistingModulePath, {
+            onBootstrapDone: () => {
+                handled = true;
+            }
+        })
+            .then(() => {
+                expect(handled).eql(true);
+                expect(errors.length).eql(1);
+                expect(errors[0].toString()).contains('Error: Cannot find module \'./non-existing-module\'');
             });
     });
 

